LoopKit/Support.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  Support.swift
+//  LoopKit
+//
+//  Created by Darin Krauss on 12/10/20.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+public protocol Support: AnyObject {
```

Comment:
Yea, this is the best name I came up with. It is basically adding additional support-related features to Loop. If you have a *much* better name, please let me know, but I'll be honest, I'm quite hesitant to change it because it is far more than a simple find/replace. There are two repos, two projects with multiple targets and multiple files, that would all require renaming.

Reply:
I understand.  I both feel strongly that this name is too vague but I also feel strongly that I shouldn't slow you down :).  Tell you what...I'll give a few suggestions below.  If you don't think any are "much" better, go ahead and stick with `Support` (this was a [nit] after all):

- `UserSupport`
- `SupportProvider`
- `SupportFacility`

Reply:
`UserSupport` isn't that bad. Now I'll just have to see if it is worth the amount of changes required.

Reply:
I kept `Support` to keep with the `*` -> `*UI` paradigm we already have. (e.g `Service` -> `ServiceUI`,`CGMManager` -> `CGMManagerUI`, `PumpManager` -> `PumpManagerUI`).

I can delete `Support`, but ultimately there will be the question "why is there no 'Support' if there is 'SupportUI'"?

Reply:
Because "Support" is something that is completely UI based. I also think it's not a separate type of service, as much as it is an optional thing that a service/device can vend.

Reply:
Okay, will do.